openreview/venue/venue.py

```diff
@@ -79,6 +79,17 @@ def get_id(self):
 
     def get_short_name(self):
         return self.short_name
+    
+    def get_edges_archive_date(self):
```

Comment:
@carlosmondra I'm using the venue.date (when the conference is happening) plus 1 year to archive the edges, let me know what you think about it.

Reply:
Seems reasonable to me